packages/vuepress-theme-nimatron/global-components/OdsTabs.vue

```diff
@@ -55,44 +55,33 @@ export default {
   },
   data() {
      return {
-      indicator: { width: 0, x: 0 },
       focusIndex: 0,
       focusCount: 0
      }
   },
-  computed: {
-    indicatorStyle() {
-      return `
-        --ods-tabs-indicator-width: ${this.indicator.width}px;
-        --ods-tabs-indicator-pos-x: ${this.indicator.x}px;
-      ` 
-    },
-  },
   methods: {
     tabSelect (event) {
       const tab = event.target
       
       this.active = tab.id
       this.focusIndex = [...tab.parentElement.children].indexOf(tab)
-      this.indicator = {
-        width: tab.offsetWidth,
-        x: tab.offsetLeft
-      }
     },
     handleTabFocus ({ key }) {
-      if (key === 'ArrowLeft') {
-        this.tabPrev()
-      }
-      else if (key === 'ArrowRight') {
-        this.tabNext()
-      }
-      else if (key === 'End') {
-        this.tabLast()
+      switch (key) {
```

Comment:
Please add a `default` case here to pick up all other key types.

Reply:
There is no reason for us to pick up any other keys. I'll admit this felt a little weird to me too, but since it was obvious we only needed to answer for those specific cases/keys I went with it. Would you prefer if I went back to a conditional?